app/assets/v2/js/cart-ethereum-polygon.js

```diff
@@ -327,7 +327,7 @@ Vue.component('grantsCartEthereumPolygon', {
 
       let networkId = appCart.$refs.cart.networkId;
 
-      if (networkId !== POLYGON_TESTNET_NETWORK_ID && networkId !== POLYGON_MAINNET_NETWORK_ID && appCart.$refs.cart.isCheckoutOngoing == true) {
+      if (networkId !== POLYGON_TESTNET_NETWORK_ID && networkId !== POLYGON_MAINNET_NETWORK_ID && appCart.$refs.cart.activeCheckout === 'standard') {
```

Comment:
Can this interfere with the zkSync checkout?
Just wondering wether we should consider:
```
appCart.$refs.cart.activeCheckout !== 'polygon'
```

Reply:
this will capture `activeCheckout === undefined` which is unintended

what we can do is

`appCart.$refs.cart.activeCheckout !== 'polygon' && appCart.$refs.cart.activeCheckout !== undefined`